### PR TITLE
T1071 T1118 Improvements and Fixes

### PR DESCRIPTION
**Pull request by apbeers**
**Details:**
Improve completion descriptions for tests. T1204 stills needs improved and the OSTap atomic tested. I just moved it over because it fits better there.

### CODE REVIEW

**Comment by apbeers** [comment]
Some of the T1071 DNS tests requires a DNS server to be set. I didn't want to put google or another one because I don't want it to look like whoever runs this is attacking externally. Someone would have to set their own to run them. Let me know if you agree with this or if we should change something on it.

File: atomics/T1071/T1071.yaml [hunk 1]
```diff
@@ -26,9 +27,9 @@ atomic_tests:
 
 - name: Malicious User Agents - CMD
   description: |
-    This test simulates an infected host beaconing to command and control.
-    Inspired by APTSimulator - https://github.com/NextronSystems/APTSimulator/blob/master/test-sets/command-and-control/malicious-user-agents.bat
+    This test simulates an infected host beaconing to command and control. Upon execution, large amounts of website source code and
```
Comment (clr2of8) [inline atomics/T1071/T1071.yaml:30]: looks like part of the sentence is missing here?

File: atomics/T1204/T1204.yaml [hunk 2]
```diff
@@ -91,7 +117,7 @@ atomic_tests:
 
 - name: OSTAP JS version
   description: |
-    Malicious JavaScript executing CMD which spaws wscript.exe //e:jscript 
+    Malicious JavaScript executing CMD which spaws wscript.exe //e:jscript
```
Comment (clr2of8) [inline atomics/T1204/T1204.yaml:120]: little typo "spaws"

**Comment by apbeers** [comment]
@clr2of8 got these fixed

**Pull request merged**